### ISSUE TITLE
Training loss 

### ISSUE THREAD

**Issue by guozixunnicolas** [Unisound/SampleRNN#16]
May I know what's the loss during your training? My loss converges around 0.5. Is that too big?
